client/src/views/Game.vue

```diff
@@ -1,65 +1,41 @@
 <template>
   <div>
     <h1 class="screen-reader-only">Planfree.dev game lobby</h1>
-
     <Modal
         v-if="modal"
         title="Choose your display name"
         @completed="enteredName"
     ></Modal>
-
-    <div v-if="!modal" class="home">
+    <Sharing v-if="showShareModal" title='share_modal_title' subTitle='share_modal_subtitle' @share="shareGame"></Sharing>
+    <div v-if="!modal && !showShareModal" class="home">
       <div class="top-buttons">
-
-
         <button class="edit-name-button" @click="modal = true">
           <div>{{ name }}</div>
           <div>
-            <svg
-                xmlns="http://www.w3.org/2000/svg"
-                height="24px"
-                viewBox="0 0 24 24"
-                width="24px"
-                fill="#000000"
-            >
-              <path d="M0 0h24v24H0V0z" fill="none"/>
+            <svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#000000">
+              <path d="M0 0h24v24H0V0z" fill="none" />
               <path
-                  d="M14.06 9.02l.92.92L5.92 19H5v-.92l9.06-9.06M17.66 3c-.25 0-.51.1-.7.29l-1.83 1.83 3.75 3.75 1.83-1.83c.39-.39.39-1.02 0-1.41l-2.34-2.34c-.2-.2-.45-.29-.71-.29zm-3.6 3.19L3 17.25V21h3.75L17.81 9.94l-3.75-3.75z"
-              />
+                d="M14.06 9.02l.92.92L5.92 19H5v-.92l9.06-9.06M17.66 3c-.25 0-.51.1-.7.29l-1.83 1.83 3.75 3.75 1.83-1.83c.39-.39.39-1.02 0-1.41l-2.34-2.34c-.2-.2-.45-.29-.71-.29zm-3.6 3.19L3 17.25V21h3.75L17.81 9.94l-3.75-3.75z" />
             </svg>
           </div>
         </button>
-        <button
-            v-if="!showCopiedToClipboard"
-            class="button invite"
-            @click="copyToClipboard()"
-        >
-          <div>Invite players</div>
+        <button v-if="!showCopiedToClipboard" class="button invite" @click="copyToClipboard()">
+          <div>{{ "Invite players"}}</div>
           <div>
-            <svg
-                xmlns="http://www.w3.org/2000/svg"
-                height="24"
-                viewBox="0 0 24 24"
-                width="24"
-            >
-              <path d="M0 0h24v24H0V0z" fill="none"/>
-              <path
-                  d="M16 1H4c-1.1 0-2 .9-2 2v14h2V3h12V1zm3 4H8c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h11c1.1 0 2-.9 2-2V7c0-1.1-.9-2-2-2zm0 16H8V7h11v14z"
-              />
+            <svg xmlns="http://www.w3.org/2000/svg" width="24px" height="24px" viewBox="0 0 24 24" version="1.1">
+              <path style=" stroke:none;fill-rule:nonzero;fill:rgb(0%,0%,0%);fill-opacity:1;"
```

Comment:
i don't love this icon, I think I'd prefer this icon:
![image](https://github.com/LukeGarrigan/planfree.dev/assets/12545967/66b1adc7-35cc-447d-a90c-264a100160ab)

Reply:
It had some additional CSS on it forcing it to be filled, I've sorted.